tests/internal/e2e/test/wdio.conf.ts

```diff
@@ -242,7 +242,7 @@ export const config: Options.Testrunner = {
   //
   // Default timeout in milliseconds for request
   // if browser driver or grid doesn"t send response
-  connectionRetryTimeout: 22_000,
+  connectionRetryTimeout: 40_000,
```

Comment:
Maybe create constant variables for all timeouts and put them together with unique names and descriptions and then use these vs hard-code

Reply:
Changed `TIMEOUT` to be imported from `vcast_utils.ts` in 27896d0687ae69b062982c784885ea857156c279.